wideq/client.py

```diff
@@ -432,7 +432,15 @@ def _get_config(self, key):
             self.device.id,
             key,
         )
-        return json.loads(base64.b64decode(data).decode('utf8'))
+        data = base64.b64decode(data).decode('utf8')
+        try:
+            return json.loads(data)
+        except json.decoder.JSONDecodeError:
+            LOGGER.warning(
+                'JSONDecodeError malformed json (%s)', data)
+            # Added fix to correct malformed JSON!
+            # see https://github.com/sampsyo/wideq/issues/64
+            return json.loads(data.replace('{[', '[').replace(']}', ']'))
```

Comment:
Can we make this a little more specific so we only attempt to replace at the beginning and end of the string?

Reply:
shall we use regex for that ? something like this works:
`return json.loads( re.sub(r'^\{(.*?)\}$', r'\1', data))`

Reply:
Yeah, seems good! Until [PEP 616](https://www.python.org/dev/peps/pep-0616/) becomes pervasive. :smiley: